setup.py

```diff
@@ -11,22 +11,16 @@
 
 # Dependencies of GPflow
 requirements = [
-    'gast==0.2.2',
     'numpy>=1.10.0',
     'scipy>=0.18.0',
-    'pandas>=0.18.1',
     'multipledispatch>=0.4.9',
-    'pytest>=3.5.0',
-    'h5py>=2.7.0',
-    'matplotlib>=2.2.2',
+    'tensorflow-probability>=0.8',
     'tabulate',
-    'tfp-nightly',
-    'dataclasses'
 ]
 
 min_tf_version = '2.0.0'
-tf_cpu = 'tf-nightly-2.0-preview'
-tf_gpu = 'tf-nightly-gpu-2.0-preview'
+tf_cpu = 'tensorflow'
```

Comment:
That's a brave move :)

Reply:
Well, it's released now, no? and it seems to work.